pkg/manifests/template/kustomize.go

```diff
@@ -38,8 +43,41 @@ func (k *kustomize) Render(svc *console.GetServiceDeploymentForAgent_ServiceDepl
 	}
 
 	command := build.NewCmdBuild(filesys.MakeFsOnDisk(), help, out)
-	path := filepath.Join(k.dir, subdir)
-	command.SetArgs([]string{path})
+	dir := filepath.Join(k.dir, subdir)
+	if err := filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() {
+			return nil
+		}
+
+		if ext := strings.ToLower(filepath.Ext(info.Name())); !lo.Contains([]string{".liquid"}, ext) {
+			return nil
+		}
+		rpath, err := filepath.Rel(dir, path)
+		if err != nil {
+			return err
+		}
+
+		data, err := os.ReadFile(path)
+		if err != nil {
+			return err
+		}
+		data, err = renderLiquid(data, svc)
+		if err != nil {
+			return fmt.Errorf("templating error in %s: %w", rpath, err)
+		}
+		if err := writeFile(path, data); err != nil {
```

Comment:
will customize choke if this file is not overwritten?  Feels like we should just `writeFile(newPath, data)` instead of overwriting the original liquid file.  This will also cause it to bork on runs that cache the tarball in the agent.

Reply:
done